src/components/search/FilterContainer.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import iconsArrowDown from '../../assets/icons/icon.svg';
+import iconsSort from '../../assets/icons/icon-sort.svg';
+//filter functionality will work after backend will fin
+const buttons = [
```

Comment:
I like how you create the array for buttons and use map method to render the button.